nemo_text_processing/inverse_text_normalization/fr/data/electronic/symbols.tsv

```diff
@@ -0,0 +1,12 @@
+chez	@
+at	@
```

Comment:
at?

Reply:
Anglicism like underscore. this one appears more regular.